mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -2570,12 +2581,20 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncImportDeletes(TALLOC_CTX *mem_ctx,
 	owner = emsmdbp_get_owner(synccontext_object);
 	openchangedb_get_MailboxReplica(emsmdbp_ctx->oc_ctx, owner, &repl_id, &replica_guid);
 
+	object_array = &request->PropertyValues.lpProps[0].value.MVbin;
```

Comment:
:hand: additional checks required. You assign a tainted data directly without prior checking if the value exists, if the index exists and if the property at this index is really of type PT_MV_BINARY:

* MS-OXCFXICS 2.2.3.2.4.5.1 states that PropertyValues (variable): An array of TaggedPropertyValue structures ([MS-OXCDATA] section 2.11.4). This value MUST NOT be NULL. -> we miss this check before assignation.

* The documentation is unclear to me whether we should expect a PidTag or just PT_MV_BINARY type and PidTag = 0x0, moreover the fixed position statement is not clear.  However as long as we have PT_MV_BINARY, you should be able to extract it using on of the convenience functions in libmapi/property.c